index.d.ts

```diff
@@ -1158,6 +1158,7 @@ declare module "eris" {
     public lastMessageID: string;
     public rateLimitPerUser: number;
     public messages: Collection<Message>;
+    public lastPinTimestamp: number | null;
```

Comment:
Isn't the `?` suffix used for this?

Reply:
I thought of it. But it seems not. Because `?` is used for optional ones. But this property always exists even if there are no pins in the channel. (returns `null` then)

Reply:
`?` seems to be used everywhere else in the typings for this purpose (a specific subset of "optional"), better to follow that convention for now